__tests__/shapes.test.js

```diff
@@ -0,0 +1,60 @@
+const PPTX = require('../index.js');
+const fs = require('fs');
+//const path = require('path');
+const tmpDir = `${__dirname}/tmp`;
+
+describe('Presentation Module', () => {
+    beforeAll(() => {
+        //prepareTmpDir(tmpDir);
```

Comment:
if we don't need the `beforeAll` or `afterAll` can we delete it?

Reply:
Yeah, I'll delete them.

Reply:
Done.